commands/Moderation/blacklist.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs-extra');
+
+exports.conf = {
+    enabled: true,
+    spamProtection: false
+};
+
+/** Run Blacklists Inhibitor
+  * @param {Object} client - The Discord.js client
+  * @param {Object} msg - The command message
+  * @param {Object} cmd - The command
+  * @returns Reject() or Resolve(), depending on if the user is blacklisted or not.
+  */
+
+exports.run = (client, msg, cmd) => {
+    return new Promise((resolve, reject) => {
+      fs.readJson(`bwd/conf/${msg.guild.id}.json`, (err, guildConf) => {

```

Comment:
You shouldn't need to use `fs` to read the confs. `client.funcs.confs` has a getter function to return a guild conf.